test/test_manifest.json

```diff
@@ -4042,6 +4042,12 @@
        "type": "eq",
        "annotations": true
     },
+    {  "id": "Bug1743245_Example",
+       "file": "pdfs/Bug1743245_Example.pdf",
```

Comment:
https://github.com/mozilla/pdf.js/pull/14460#discussion_r791089822 still applies here, and to the `.gitignore` file.

Reply:
I changed the file names to the naming convention, as mentioned.